<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>541</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutCentral">
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>B Koodak</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFont"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFontSize"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxRevWord">
         <property name="text">
          <string>Revese Words</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxRevLetter">
         <property name="text">
          <string>Revese Letters</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0340909 rgba(121, 121, 121, 255), stop:1 rgba(36, 36, 36, 255));</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{ 
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, 		stop:0 rgba(111, 111, 111, 255), stop:0.352273 rgba(120, 120, 120, 255), stop:0.681818 rgba(133, 133, 133, 255), stop:1 rgba(108, 108, 108, 255));
}
QPushButton:pressed{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(93, 93, 93, 255), stop:0.352273 rgba(109, 109, 109, 255), stop:0.6875 rgba(117, 117, 117, 255), stop:1 rgba(93, 93, 93, 255));
 }

QPushButton{
	color: rgb(255, 255, 255);
	border-color: rgb(255, 0, 4);
	border:1px solid black;
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(97, 97, 97, 255), stop:0.352273 rgba(106, 106, 106, 255), stop:0.681818 rgba(118, 118, 118, 255), stop:1 rgba(91, 91, 91, 255));
	selection-background-color: rgb(181, 144, 255);
	font: 75 8pt &quot;Tahoma&quot;;font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Copy to clipboard</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionUnicode"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionUnicode_File"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSetting"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/folder_empty.png</normaloff>:/icon/icon/folder_empty.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/application.png</normaloff>:/icon/icon/application.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCharectrs">
   <property name="text">
    <string>Charectrs</string>
   </property>
  </action>
  <action name="actionUnicode">
   <property name="text">
    <string>Unicode</string>
   </property>
  </action>
  <action name="actionUnicode_File">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/calendar.png</normaloff>:/icon/icon/calendar.png</iconset>
   </property>
   <property name="text">
    <string>Unicode File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
